### ISSUE TITLE
XeSS 1.2.0 and 1.3.1 not supported in games

### ISSUE THREAD

**Issue by stalker-svk** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#879]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game [Required]

Hogwarts Legacy

### Game Platform [Required]

- [X] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Amazon Games app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

i5-13500

### Graphic Card [Required]

A770 16GB

### GPU Driver Version [Required]

32.0.101.6319

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build [Required]

Windows 11 23H2

### Intel System Support Utility report

[scan.txt](https://github.com/user-attachments/files/18106811/scan.txt)

### Description and steps to reproduce [Required]

XeSS can't be enabled in games that should support it, for example Hogwarts Legacy, S.T.A.L.K.E.R. 2, also in 3DMark XeSS test only versions 1.0.0 and 1.1.0 works, newer 1.2.0 and 1.3.1 do not work
![Capture](https://github.com/user-attachments/assets/72105150-8582-4b50-a605-f3f9d95ca8ec)
![Capture2](https://github.com/user-attachments/assets/b4184243-3249-49fd-9433-12fd957977f8)
![Capture3](https://github.com/user-attachments/assets/4d6e802a-8416-4d70-b2e5-f9ca0a9165ea)
![Screenshot (300)](https://github.com/user-attachments/assets/7880fbfe-8e8b-4bf7-8fe4-fd0f0c9a264c)

### Game graphic quality [Required]

- [ ] Low
- [ ] Medium
- [ ] High
- [ ] Ultra

### Game resolution [Required]

1920x1080

### Game VSync [Required]

Off

### Game display mode [Required]

- [X] Fullscreen
- [X] Windowed
- [X] Borderless Windowed

### Detailed game settings [Required]

![Screenshot (300)](https://github.com/user-attachments/assets/702c2342-4aa9-40c6-97c2-eacd7d964121)

**Comment by stalker-svk** [comment]
yes I tried DDU in safe mode, then installed newest driver but no success

**Comment by stalker-svk** [comment]
yes I have it and that file is in that folder but even XeSS SDK 1.3.1 demo shows that it is unsupported so it's not problem of one game

**Comment by Ilya-intel** [comment]
@stalker-svk 
Did it work before? Or did it start at some point in time?

**Comment by stalker-svk** [comment]
it worked when Hogwarts Legacy was using older version, after they came with update that brought XeSS 1.3, it disappeared from setting, that time it was the only XeSS game I had...I thought they removed support but now when I see that every game have problem with it I started to troubleshoot it

**Comment by Ilya-intel** [comment]
@stalker-svk 
Looks like something happened to OS system files. 
Did you try command sfc /scannow to check system files integrity?

**Comment by stalker-svk** [comment]
tried it right now but no success, still no XeS 1.2.0 and newer

**Comment by stalker-svk** [comment]
I noticed that older Xess versions (1.0.0 to 1.1.0) have more DLLs (igxess, libxess, XeFX and XeFX_Loader) and newer versions (1.2.0 to 1.3.1) have only 1 DLL named libxess.dll. Can this be the problem? do i have to manually register that DLL?

**Comment by stalker-svk** [comment]
when I try to manually register the DLL from 1.3.1 SDK using regsvr32 it gives me that error
![Capture](https://github.com/user-attachments/assets/92dd48d0-1c58-4e51-8037-89a312767b40)

**Comment by Ilya-intel** [comment]
@stalker-svk 
Newer implementations of XeSS require only one dll, this is correct.
And you don't need to register it, the game loads and initializes it during launch.
There is something else...

**Comment by stalker-svk** [comment]
XeSS 1.3.1 SDK log shows I have unsupported driver, I'm using newest driver and always choose clean install when installing newer driver, driver updates via windows update are disabled by group policy to prevent messing with drivers
![Capture](https://github.com/user-attachments/assets/6e8fc1ee-a971-44b6-baea-4937b771d627)

**Comment by stalker-svk** [comment]
I read that XeSS is disabled in windows insider preview versions but I'm not enrolled in insider program (and I never was)

**Comment by Ilya-intel** [comment]
@stalker-svk 
Do you have any sort of third-party GPU management tools like MSI Afterburner?
Also, could you pull out the report from the DXDiag tool? Just press the button "Save All Information Button" for text file to be generated.

**Comment by stalker-svk** [comment]
here is DxDiag saved file and windows build number.
![Capture](https://github.com/user-attachments/assets/d7810821-00d2-4efe-a527-5d2677e202a6)
[DxDiag.txt](https://github.com/user-attachments/files/18209558/DxDiag.txt)

**Comment by Ilya-intel** [comment]
@stalker-svk 
I don't see anything unusual in the logs.
So, as a matter of troubleshooting, could you update your BIOS to 1805 and then do a DDU in safe mode and install gfx driver from [here](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html)?

**Comment by stalker-svk** [comment]
problem still persists :( updated BIOS and did DDU in safe mode, then installed newest driver but still no XeSS
[DxDiag.txt](https://github.com/user-attachments/files/18230026/DxDiag.txt)

**Comment by Ilya-intel** [comment]
@stalker-svk 
Ok, couple more things to try:
1. Uninstall dxtory device and drivers.
2. Plug in display via HDMI, do a clean driver installation without DDU, just choose "clean" during driver installation.

**Comment by stalker-svk** [comment]
still same problem.....maybe there is something corrupted in my OS but I want to avoid reinstallation of windows because I have done many customizations and don't want to do it all again...maybe I try install another windows 11 on external disk just to test the XESS and try to reproduce what caused it to disappear

**Comment by SavageWaffle** [comment]
Op this is likely a OS configuration issue specific to you unfortunately, does it persist if you disable IGPU in BIOS? It seems like games are reading that potentially and not seeing the XeSS compatible gpu?

**Comment by stalker-svk** [comment]
I don't use iGPU, it's always disabled, monitor is connected to dGPU

**Comment by SavageWaffle** [comment]
I understand but can you test and see if any difference occurs with actually disabling onboard graphics via BIOS settings? Unless you are saying you have explicitly disabled it already to no avail.

**Comment by stalker-svk** [comment]
integrated graphics IS disabled automatically when you connect monitor into dedicated graphics, it even don't show in BIOS

**Comment by SavageWaffle** [comment]
I noticed that too, you think AMD would do something similar but alas they do not. I think you may need to test with either updating your Windows 11 to 24H2 and or reinstalling it fresh and retesting on a clean system image post driver installs?

**Comment by stalker-svk** [comment]
I don't want to do clean install of windows because I have done so many customizations and I don't want to do it again, maybe I try to make backup image and upgrade to 24H2 or install second windows 11 on external drive and try to reproduce what caused this problem and then backtrack the steps to restore it to working state

**Comment by SavageWaffle** [comment]
Good idea, if the issues don’t persists in a clean image then we know that you’re customizations simply have broken something within windows and the graphics driver functionality? Keep us posted once the next driver drops this week.

**Comment by SavageWaffle** [comment]
@stalker-svk New driver help any?

**Comment by stalker-svk** [comment]
> [@stalker-svk](https://github.com/stalker-svk) New driver help any?

no

**Comment by SavageWaffle** [comment]
@stalker-svk :( Tough one @Ilya-intel hopefully can reproduce but it may be a isolated issue if not.

**Comment by Ilya-intel** [comment]
Hey @stalker-svk ,
Any luck testing on another Windows installation?
Also, the Xess Inspector tool was updated, could you try it? https://github.com/GameTechDev/XeSSInspector

**Comment by stalker-svk** [comment]
did not tested it on another installation but after update to 24H2 still same problem...and I'm getting tired of it, my next GPU won't be intel one, for issues like this, I want fully working solution that don't need any tweaks and workarounds, when I launch a game that should support XeSS, that inspector app shows no xess context found

**Comment by Ilya-intel** [comment]
@stalker-svk 
We're not seeing such reports from other users and couldn't replicate it on several builds, so it's most likely an OS-specific issue.

**Comment by stalker-svk** [comment]
I did clean install of Windows 11 24H2 and now XeSS is working again, here is the proof 

![Image](https://github.com/user-attachments/assets/3f53d670-e343-407e-85c9-c6a1aa2eda87)

but when I install this update:

![Image](https://github.com/user-attachments/assets/fdcce57b-b2ab-4462-8b42-358c76cba972)
the same error about unsupported driver appears, that update is marked as Preview so can you please check it on your side? I'm not in Insider program, I'm using public build of windows 11 24H2, maybe this update marked as "preview" makes XeSS think it's an Insider build (because I read somewhere that XeSS does not work in Insider previews)

**Comment by Ilya-intel** [comment]
@stalker-svk 
Interesting, let me check this update and get back to you with findings.

**Comment by stalker-svk** [comment]
finally found the main cause of the problem and it was that I have disabled these 2 tasks in Task Scheduler:

![Image](https://github.com/user-attachments/assets/d89b6f7d-83a4-4b9c-81d3-23f9d9615ceb)
you can try it on your side to reproduce the error, disable them and reboot, now the "unsupported driver" error will pop...re-enable those tasks and reboot and XeSS is back

and why I'm disabling tasks? I want to strip down windows as much as possible while keeping it working without any errors and problems, so part of my tweaks is disabling tasks in task scheduler and disabling unneeded windows services.
so the problem was at my side, not at you, you can close this issue and I'm very sorry for blaming you for this issue

**Comment by stalker-svk** [comment]
it is also back in games

![Image](https://github.com/user-attachments/assets/1565ba54-0781-4fed-8d50-e9e9e14b832b)
